netcdf_to_cb/test/test_int_metar_obs_netcdf.py

```diff
@@ -25,13 +25,15 @@ def setup_connection():
 def test_one_thread_specify_file_pattern():  # pylint:disable=missing-function-docstring
     try:
         # setup - remove output files
+        if not os.path.exists('/opt/data/netcdf_to_cb/output/test1'):
+            os.makedirs('/opt/data/netcdf_to_cb/output/test1')
         for _f in glob("/opt/data/netcdf_to_cb/output/test1/*.json"):
             os.remove(_f)
         vx_ingest = VXIngest()
         vx_ingest.runit(
             {
-                "job_id": "JOB:V01:METAR:NETCDF:OBS",
-                "credentials_file": os.environ["HOME"] + "/adb-cb1-credentials",
+                "job_id": "JOB-TEST:V01:METAR:NETCDF:OBS",
+                "credentials_file": os.environ["CREDENTIALS"],
                 "path": "/opt/data/netcdf_to_cb/input_files",
```

Comment:
Similar in spirit to the pytest `tmp_dir` fixture comment - could this `path` variable be made configurable using one of the env variables we require so that we can define where the test data is?

Reply:
sure

Reply:
Actually I think the path parameter should be totally removed and the path always specified in a JOB-TEST job document. That is a lot cleaner. I removed the path parameter from the runit methods for both the grib_builder and netcdf_builder VxIngest classes..

Reply:
Agreed,  if the path should be in the job doc, that'll definitely clean things up!